pylops/optimization/solver.py

```diff
@@ -273,6 +273,9 @@ def lsqr(
     Solve an overdetermined system of equations given an operator ``Op`` and
     data ``y`` using LSQR iterations.
 
+    .. math::

```

Comment:
What is this?

Reply:
It declares `\cond\` as a math operator (like `\max`, `\tan`, etc.). Since "cond" is used quite a bit in the documentation, I just declared it there. It does not change the rendering at that location (the LaTeX command is hidden).